src/components/typography/typography.js

```diff
@@ -0,0 +1,21 @@
+import React from "react"
+import style from "./typography.module.scss"
+import cn from "classnames"
```

Comment:
Just a minor thingy about code style:
I propose we could order the imports like this:
1. React and any other "global" libraries
2. local javascript modules
3. any styles/images/fonts